webtool/test_transcribe.py

```
@@ -1672,10 +1679,98 @@ def platzt(*_a, **_kw):
+    ersten Zeile feststand.
+
+    BEIDE Richtungen, sonst ist die halbe Regel unbewacht — die zweite ist die teurere: ohne
+    Anbieter ist die LLM-Phase bewusst aus, und ein absichtlich ausgelassener Schritt darf den
+    Lauf nicht rot faerben (dieselbe Regel wie beim Kill-Switch).
+    """
+    from webtool import correct, llm
+
+    # (1) Pool steht: gescheiterter VERSUCH — Zeile UND Zaehler.
+    _ki_projekt(monkeypatch, tmp_path, "PrepFalschDemo")
+    monkeypatch.setattr(llm, "available", lambda: (True, ""))
+    monkeypatch.setattr(correct, "cmd_diarize", lambda *a, **kw: None)
```

Ruff (0.16.2)

[warning] Unused lambda argument: `a`

(ARG005)